caikit/runtime/servicers/global_predict_servicer.py

```diff
@@ -88,6 +88,8 @@ class GlobalPredictServicer:
     given request
     """
 
+    MODEL_MESH_MODEL_ID_KEY = MODEL_MESH_MODEL_ID_KEY
```

Comment:
Not necessary?

Reply:
Good catch! That was from when I renamed everything in this branch before making [this PR](https://github.com/caikit/caikit/pull/634)